src/components/Calendar/Calendar.tsx

```diff
@@ -30,14 +30,19 @@ export interface CalendarEvent {
 }
 
 const CalendarStyling = styled.div`
-    --fc-button-active-bg-color: ${(props) => props.theme.secondaryBackgroundColor};
-    --fc-button-active-border-color: ${(props) => props.theme.secondaryBackgroundColor};
-    --fc-button-text-color: ${(props) => props.theme.secondaryForegroundColor};
+    --fc-button-active-bg-color: ${(props) => props.theme.primary.background[3]};
+    --fc-button-active-border-color: ${(props) => props.theme.primary.background[3]};
+    --fc-button-text-color: ${(props) => props.theme.primary.foreground[3]};
```

Comment:
Still think this isn't immediately obvious what [3] means, but if everyone else is happy with it that is obvs fine :)

Reply:
2 is the middle and the palette ranges from 0 to 4 
So in light mode - 3 is the darker color from the usual green you can see everywhere (it just means the active button is darker than usual green!)